src/views/red-experience.njk

```diff
@@ -0,0 +1,126 @@
+      </h4>
+      {{ govukInput({
+        classes: "govuk-input--width-4",
+        id: "red-control-number",
+        name: "redControlNumber",
+        errorMessage: {
+          text: 'The number of red deer you control must only contain numbers'
+        } if model.redControlNumberInvalidError
+      }) }}
+  {% endset -%}
+
+  {% if model.experienceError %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+            text: "You must select yes or no",
+            href: "#red-experience-years-error"
+        } if model.redSelectionError,
+        {
+            text: "You must enter the number of years experience you have controlling red deer",
+            href: "#red-experience-years-error"
+        } if model.redEmptyError,
+        {
+            text: "The number of years experience you have controlling red deer must only contain numbers",
+            href: "#red-experience-years-error"
+        } if model.redExperienceYearsInvalidError,
+        {
+            text: "The number of red deer you control must only contain numbers",
+            href: "#red-control-number-error"
+        } if model.redControlNumberInvalidError
+      ]
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+    <form method="post">
+      {{ govukRadios({
+        id: "red",
+        name: "red",
+        fieldset: {
+          legend: {
+            text: "Do you control red deer?",
+            isPageHeading: true,
+            classes: "govuk-fieldset__legend--l"
+          }
+        },
+        errorMessage: {
+          text: "You must select yes or no"
+        } if model.redSelectionError,
+        items: [
+          {
+            value: "yes",
+            text: "Yes",
+            conditional: {
+              html: yesHtml
+            },
+            checked: model.red === 'yes'
+          },
+          {
+            value: "no",
+            text: "No",
+            checked: model.red === 'no'
+          }
+        ]
+      }) }}
+
+      {{ govukButton({
+      text: "Continue",
+      classes: "naturescot-forward-button"
+      }) }}
+    </form>
+
+      
+      <h3 class="govuk-heading-s">Email</h3>
+      <ul class="govuk-list">
+        <li><a class="govuk-link" href="mailto:licensing@nature.scot">licensing@nature.scot</a></li>
+      </ul>
+      <h3 class="govuk-heading-s">Telephone</h3>
+      <ul class="govuk-list">
+        <li>01463 725 364</li>
+        <li>Monday to Friday, 10am to 4pm (except public holidays)</li>
+      </ul>
+      <p class="govuk-body">
+        <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="https://www.gov.uk/call-charges">Find out about call charges&nbsp;(opens in new tab)</a>
+      </p>
+      <a class="govuk-link govuk-link--no-visited-state govuk-!-font-size-16" href="#top-of-page">
+        Back to top
+      </a>
```

Comment:
I also just noticed this.

I think you've just copied and pasted from a previous page and included too much. I've done a quick check and it's on quite a few pages, so I'll make a new card to address all the pages in one go.